arm_robots/src/arm_robots/cartesian.py

```diff
@@ -0,0 +1,221 @@
+import copy
+import enum
+
+import numpy as np
+import rospy
+from geometry_msgs.msg import PoseStamped
+from victor_hardware_interface_msgs.msg import ControlMode, MotionCommand
+
+
+def point_to_np_array(a):
+    return np.array([a.x, a.y, a.z])
+
+
+def pose_distance(a, b):
```

Comment:
this does a position-only not really a pose distance, perhaps rename?

Reply:
Actually does compute pose distance now

Reply:
Actually does pose distance now